make_level.py

```diff
@@ -0,0 +1,117 @@
+            level_setup = "\n".join([
+                "import pkg_resources\n",
+                "from gitgud.skills.util import BasicLevel",
+                "from gitgud.skills.util import Skill\n"
+                "skill = Skill(",
+                "    '{}',".format(skill_name),
+                "    [",
+                "    ]",
+                ")\n",
+                ])
+            fp.write(level_setup)
+    
+        
+        # Read file
+        with open(os.path.join(os.path.join("gitgud","skills"),"__init__.py"), 'r') as fp:
+            filedata = fp.read()
+
+        # Add import statement
+        replace = "from gitgud.skills.{0} import skill as {0}_skill\n\nfrom gitgud.skills.u".format(skill_name)
+        filedata = filedata.replace("\nfrom gitgud.skills.u", replace)
+        
+        # Add to input array of Skill
+        replace = ",\n    {}_skill\n]".format(skill_name)
+        filedata = filedata.replace("\n]", replace)
+
+        # Write to file
+        filepath = os.path.join(os.path.join("gitgud","skills"),"__init__.py")
+        with open(filepath, 'w') as fp:
+            fp.write(filedata)
+            print("Modified: Added skill to {}".format(filepath))
+        
+        write_init(skill_name, skill_path, level_name)
+        return
+    else:
+        # Populate file with BasicLevel
+        with open(os.path.join(skill_path,"__init__.py"), 'r') as fp:
+            filedata = fp.read()
+
+        replace = ",\n        BasicLevel('{0}', pkg_resources.resource_filename(__name__, '_{0}/'))\n    ]".format(level_name)
+        filedata = filedata.replace("\n    ]", replace)
+        filedata = filedata.replace("[,","[")
+        
+        filepath = os.path.join(skill_path,"__init__.py")
+        with open(filepath, 'w') as fp:
+            fp.write(filedata)
+            print("Modified: Added level to {}".format(filepath))
+    return
+
+def create_file(level_path,filename):
+    with open(os.path.join(level_path,filename),'a+'):
+        pass
+    print("Created: {}".format(filename))
+
+def main():
+    # Obtain input arguments
+    skill_name = sys.argv[1]
+    level_name = sys.argv[2]
+    
+    # Check if cwd is gitgud folder
+    if os.path.isdir(os.path.join(cwd, 'gitgud')):
+            # Confirm choice to avoid making a mess
+            print ("\n".join([
+                            "skill_name: {}".format(skill_name),
+                            "level_name: {}".format(level_name),
+                            "Confirm[y/n] "
+                            ]),
+                        end = '')
+            choice = input().lower()
+            if choice == 'n':
+                return
+            
+            print("\nCreating Folders:")
+            # Make skill folder
+            skill_path = os.path.join("gitgud","skills","{}".format(skill_name))
+            if not os.path.exists(skill_path):
+                os.mkdir(skill_path)
+                print("Created: {}".format(skill_name))
```

Comment:
Let's output the path, so 
Created: "gitgud/skills/foo/"
Created: "gitgud/skills/foo/_bar/"

Reply:
Reflected in next push